core/src/main/java/org/phoenixframework/liveview/lib/Document.kt

```diff
@@ -1,17 +1,25 @@
 package org.phoenixframework.liveview.lib
 
+
 class Document {
     private var nativeObject: Long
+    private var asBorrow: Boolean
```

Comment:
This can't change outside of the constructors, so it can be `val` instead of `var`

```suggestion
    private val asBorrow: Boolean
```